content/en/users/compute/orchestration/im/dashboard/_index.md

```diff
@@ -148,38 +148,48 @@ infrastructures:
 
 - Start: Starts/Resumes a previously stopped infrastructure.
 
-- Outputs: Shows the outputs of the TOSCA template. Private key of
-  credentials can be downloaded as a file or copied to the clipboard.
+- Outputs: Shows the outputs of the TOSCA template. Private key of credentials
+  can be downloaded as a file or copied to the clipboard.
 
-   ![TOSCA outputs](dash_outputs.png)
+  ![TOSCA outputs](dash_outputs.png)
 
-- Delete: Delete this infrastructure and all the asociated resources. It also
-  has the option to force the deletion. In this case the infrastructure will
-  be removed from the IM service even if some cloud resources cannot be deleted.
+- Delete: Delete this infrastructure and all the associated resources. It also
+  has the option to force the deletion. In this case the infrastructure will be
+  removed from the IM service even if some cloud resources cannot be deleted.
   **Only use this option if you know what you are doing**.
 
-- Delete & Recreate: Delete this infrastructure as the previous option, but
-  once it is deleted it will redirect to the infrastructure creation form, with
-  all the input fields filled with the same set of values used to create the
-  deleted infrastructure.
+- Delete & Recreate: Delete this infrastructure as the previous option, but once
+  it is deleted it will redirect to the infrastructure creation form, with all
+  the input fields filled with the same set of values used to create the deleted
+  infrastructure.
 
 - Reconfigure: Starts the reconfiguration of the infrastructure.
 
+- Change User: Add or change the ownership of the infrastructure at IM level.
+  Providing a valid `Access Token` of another user, the infrastructure can be
+  shared or transferred to them. If `Overwrite` is checked, the new user will be
+  the unique owner of the infrastructure (transferring), otherwise it will be
+  added to the list of current users (sharing). **The new user must be member of
+  the VO used to create the resources**, otherwise he will not be able to manage
+  them.
+
+  ![Change/Add User modal form](dash_change_user.png)
+
 ### VM Info page
 
-The VM Info page shows all the information about the selected VM and
-enables to manage the lifecycle of it. On the top right corner the `Manage VM`
-dropdown menu allows to Stop/Start, Reboot, Resize, Reconfigure and Terminate
-the VM. Furthermore the user can check the error or contextualisation log of this
+The VM Info page shows all the information about the selected VM and enables to
+manage the life cycle of it. On the top right corner the `Manage VM` drop-down
```

Comment:
```suggestion
manage the lifecycle of it. On the top right corner the `Manage VM` drop-down
```
So the spell checked wanted life cyle but the terminology linter prefers lifecycle, let's agree with him.